cas-server-documentation/protocol/CAS-Protocol-Specification.md

```diff
@@ -558,7 +558,9 @@ are case sensitive and MUST all be handled by `/serviceValidate`.
     primary credentials. It will fail if the ticket was issued from a single
     sign-on session.
 
-
+-   `format` [OPTIONAL] - if this parameter is set, ticket validation response
```

Comment:
I don't know yet, but I was trying to anticipate some kind of anticipation. Maybe it's too much speculation.